eve/main.yml

```diff
@@ -2537,14 +2537,6 @@ stages:
           name: Import Solution next example-solution archive
           env:
             SOLUTION_ARCHIVE: /var/tmp/example-solution-next.iso
```

Comment:
Oh btw you also want to remove the 2 imports right before that are only used for this cypress tests (sorry I forgot :smile: )

Reply:
Hi @teddy!

Thanks for the double-check!
https://github.com/scality/metalk8s/pull/3167/commits/55b1e01da3dc85430493501c5ba648006b3d993b